pyproject.toml

```diff
@@ -36,3 +36,6 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
 mobilizon-reshare="mobilizon_reshare.cli.cli:mobilizon_reshare"
+
+[tool.pytest.ini_options]
+addopts = "--cov-config=.coveragerc --cov=. --cov-report html tests/"
```

Comment:
io metterei "term-missing" invece che html come default. Così non sta a creare file e la vede anche un utente che non sa che è configurato, senza doverselo andare a cercare.

Reply:
ho capito, sì in effetti può essere una buona idea.... come funziona? Devo fare una nuova PR con la modifica ?

A prescindere da questo : che tu sappia poi è facilmente rimpiazzabile o estendibile il solo argomento --cov-report senza dover riscrivere tutta la riga quando si ha bisogno di un report html? Almeno quello è abbastanza completo e ti mostra/evidenzia proprio per intero le righe mancanti all'interno dei listati e non solo i relativi numeri.

Reply:
credo i parametri da CLI sovrascrivano. Comunque no, pusha sullo stesso branch e basta, la PR è da branch a branch, non da commit a commit

Reply:
Ok, ho fatto un nuovo commit sul mio branch con l'opzione cambiata da HTML a term-missing

Reply:
non hai pushato però